packages/sdks/src/functions/get-processed-block.ts

```diff
@@ -20,11 +45,14 @@ const evaluateBindings = ({
   if (!block.bindings) {
     return block;
   }
-  const copy = fastClone(block);
+  // TODO: don't fast clone this! Any
+  const copy = deepCloneWithConditions(omit(block, 'children', 'meta'));
```

Comment:
@steve8708 can you expand on the reasoning behind this? does it improve perf? or solve a bug?

Reply:
yeah each block can have infinite child blocks, we don't want to re-clone each child when we visit each child, so this is to improve perf. 

tl;dr we want to clone basically everytihng except blocks. so if we ever hit a block we bail and return the block itself without traversing deeper

we also skip meta bc some people put a lot of stuff in meta and we don't technically claim to support bindings in meta (tehre is no point of that, meta isn't used in rendering)

so the idea is really lets just clone properties, styles, and component options (that aren't blocks)